src/Testing/TypeInferenceTestCase.php

```diff
@@ -141,8 +144,14 @@ public function assertFileAsserts(
 	 */
 	public static function gatherAssertTypes(string $file): array
 	{
+		$currentWorkingDirectory = getcwd();
+		if ($currentWorkingDirectory === false) {
+			throw new ShouldNotHappenException();
+		}
+		$pathHelper = new SimpleRelativePathHelper($currentWorkingDirectory);
```

Comment:
Isn't this already available in the container?

Reply:
The CWD parameter for sure, maybe even this exact helper too.

Reply:
I'm worried this might be the fuzzy one? I think you're supposed to get the right one by name (different Container method). See config.neon.